### PR TITLE
R rstudio

### PR DESCRIPTION
**Pull request by annakrystalli**
Add final version of r rstudio content. Have removed introductory sections that we will run through together

### CODE REVIEW

File: content/04-r-rstudio/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "R & Rstudio"
+title = "Cloning and Collaborating with R & Rstudio"
```
Comment (davidwyld) [inline content/04-r-rstudio/_index.md]: I got rid of 'cloning and collaborating with' from the other chapters, do you want this in?

**Review by davidwyld — APPROVED** [review]
looks great! nice one! busy afternoon?

File: content/04-r-rstudio/_index.md (hunk 1 above)
Comment (annakrystalli) [reply content/04-r-rstudio/_index.md]: That's just the chapter landing page. All chapters require one. It's not got any info in it

Comment (annakrystalli) [reply content/04-r-rstudio/_index.md]: Oh sorry!! Completely misread that comment!

**Pull request merged**